src/conic_form.jl

```diff
@@ -0,0 +1,270 @@
+mutable struct ConeData
+    f::Int # number of linear equality constraints
+    l::Int # length of LP cone
+    q::Int # length of SOC cone
+    qa::Vector{Int} # array of second-order cone constraints
+    s::Int # length of SD cone
+    sa::Vector{Int} # array of semi-definite constraints
+    ep::Int # number of primal exponential cone triples
+    ed::Int # number of dual exponential cone triples
+    p::Vector{Float64} # array of power cone params
```

Comment:
Could we move `Float64` in `ConeData` template? Something like
`mutable struct ConeData{T}`
Also, do we want `ConeData` to be mutable?

Reply:
done